internal/worker/sshserver/worker.go

```diff
@@ -115,9 +116,22 @@ func (ssw *serverWrapperWorker) loop() error {
 
 	port := config.SSHServerPort()
 
+	// TODO(ale8k): As of right now, the generated host key is in mongo.
+	// The initialisation logic needs migrating over to DQLite and then
+	// a domain service should call the method to retrieve the generated
+	// host key here. For now, we'll generate it to stop the server bouncing.
+	//
+	// We could access state directly in the worker here, but 4.0 isn't
+	// released yet anyway.
+	hostKey, err := pkissh.NewMarshalledED25519()
```

Comment:
I'm probably missing some of the background context, the ssh worker won't work correctly with this key right?
If that's the case, why not simply change the worker so it doesn't return an error in this specific case? I ask because in any case it's broken.

Reply:
It will, it'll just generate a host key on startup. If the worker restarts, it'll have a new one. But this is just an interim solution until change over the initialise() logic to be DQLite and implement a domain service. I'm just saving time here to focus time and efforts on the more pressing 3.6 work.